composer.json

```diff
@@ -0,0 +1,27 @@
+{
+    "name": "davidandvinai/preferencesinfocommand",
```

Comment:
I am up for that. what about adding in `-` between words to make it a bit "easier" to read. Not sure myself?